freebasics/static/css/template-base.css

```diff
@@ -210,16 +210,25 @@ h6{
 .flatbutton-input{
 	-moz-border-radius:2px;
 	-webkit-border-radius:2px;
-	border-radius:2px;
-	display:inherit;
+  background-color: #dfdfdf;
+  font-family:"Bangers", Helvetica, sans-serif;,
+  text-transform: uppercase;
+  border-radius: 3px 3px 3px 3px;
+	display:inline;
 	cursor:pointer;
-	color:#fff;
 	font-size: 12px;
 	padding: 8px 22px;
 	text-decoration:none;
 	border: 15px;
   text-align: center;
 }
+.flatbutton-input:hover {
+	background-color:#c736c0;
```

Comment:
Why are you hardcoding this purple button. I thought we were going to use `accent 1`